apps/modernjs-ssr/dynamic-nested-remote/modern.config.ts

```diff
@@ -0,0 +1,22 @@
+import { appTools, defineConfig } from '@modern-js/app-tools';
+import { moduleFederationPlugin } from '@module-federation/modern-js';
+
+// https://modernjs.dev/en/configure/app/usage
+export default defineConfig({
+  dev: {
+    port: 3009,
+  },
+  runtime: {
+    router: true,
+  },
+  output: {
+    disableTsChecker: true,
```

Comment:
Why is ts check turned off

Reply:
It can be removed now , i added it for test